src/NUnitFramework/framework/Attributes/TestCaseSourceAttribute.cs

```diff
@@ -254,6 +242,16 @@ private IEnumerable<ITestCaseData> GetTestCasesFor(IMethodInfo method)
             return data;
         }
 
+        private static object[] TransposeArray(Array arguments)
```

Comment:
The name is odd for a method that basically copies one array to another. The language doesn't really have a notion of vertical versus horizontal.